internal/daysteps/daysteps.go

```diff
@@ -1,15 +1,49 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+	"go-4-sprit-final/internal/spentcalories"
+	"strconv"
+	"strings"
 	"time"
 )
 
+const (
+	STEP_LENGTH = 0.65 // длина шага в метрах
+)
+
 var (
-	StepLength = 0.65 // длина шага в метрах
+	ErrorConvData =  errors.New("this data isn't correction conv")
+	ErrorCorrectionData =  errors.New("this data isn't correction")
+	ErrorNullStepData =  errors.New("steps = 0")
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// ваш код ниже
+	stepsAndTime := strings.Split(data, ",")
+
+	if len(stepsAndTime) != 2{
+		return 0, 0, ErrorCorrectionData
+	}
+
+	steps, err := strconv.Atoi(stepsAndTime[0])
+
+	if err != nil{
+		return 0, 0, ErrorConvData
```

Comment:
1. Теряешь сообщение об ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```